Reemit.Decompiler.Clr/Disassembler/InstructionComparer.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace Reemit.Decompiler.Clr.Disassembler;
+
+public class InstructionComparer : IEqualityComparer<Instruction>
+{
+    public static InstructionComparer Instance = new InstructionComparer();
+
+    public bool Equals(Instruction? x, Instruction? y) =>
+        OpcodeInfoComparer.Instance.Equals(x?.OpcodeInfo, y?.OpcodeInfo) &&
+        OperandComparer.Instance.Equals(x?.Operand, y?.Operand);
+
+    public int GetHashCode([DisallowNull] Instruction obj)
```

Comment:
Looks like we have a redundant attribute here:
<img width="646" alt="image" src="https://github.com/user-attachments/assets/0ac9e791-f36d-4ca6-b67e-9579446ca187">

Reply:
Done